poof.py

```diff
@@ -62,10 +62,13 @@ def __call__(self, params=None):
     def run(self, args):
         p = Popen(args, stdout=PIPE, stderr=PIPE)
         out, err = p.communicate()
+        out, err = out.strip(), err.strip()
+        if sys.version_info >= (3, 0):
+            out, err = str(out, 'utf-8'), str(err, 'utf-8')
         if p.returncode == 0:
```

Comment:
This is not the desired behavior: if there isn't any error (return code is 0) then returns the terminal output in `out`, otherwise returns the message error in `err`. It is `(True, out)` xor `(False, err)`.